src/i18n/strings/bg.json

```diff
@@ -840,7 +840,6 @@
     "Download this file": "Изтегли този файл",
     "Pin Message": "Закачи съобщението",
     "Failed to change settings": "Неуспешна промяна на настройки",
-    "View Community": "Прегледай общността",
```

Comment:
We don't remove translations in this way - just run the `yarn i18n` script instead and it'll do the needful for now.

Reply:
OK, I just removed all of the changes to src/i18n except the one done automatically by `yarn i18n`.